app/uk/gov/hmrc/thirdpartyorchestrator/commands/applications/domain/models/DispatchSuccessResult.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartyorchestrator.commands.applications.domain.models
+
+import play.api.libs.json.Json
+
+import uk.gov.hmrc.apiplatform.modules.applications.core.domain.models.ApplicationResponse
+
+case class DispatchSuccessResult(applicationResponse: ApplicationResponse)
+
+object DispatchSuccessResult {
```

Comment:
Probably need to sort what TPA returns in that most LDT are as millis except TPOUA.